packages/edit-site/src/components/editor/index.js

```diff
@@ -94,15 +95,21 @@ export default function Editor() {
 				'core/edit-site',
 				'showIconLabels'
 			),
+			showBlockBreadcrumbs: select( editSiteStore ).isFeatureActive(
+				'showBlockBreadcrumbs'
+			),
```

Comment:
`isFeatureActive` is deprecated, so it'd be good to match the line above and use the preferences store. 

```suggestion
			showBlockBreadcrumbs: select( preferencesStore ).get(
			    'core/edit-site',
			    'showBlockBreadcrumbs'
			),
```

Reply:
Good catch, thanks! 🤦‍♂️